services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java

```diff
@@ -115,11 +115,6 @@ void shouldNotThrowExceptionWhenValidateForPpaIsMoreThan23HoursSameDay() {
     TimeUtils.setNow(null);
   }
 
-  @Test
-  void testAllRelevantSrsCases() {
-    //FIXME: Implement tests for SRS
```

Comment:
the SRS tests are already implemented

Reply:
Where are they implemented? If so, we should just remove this method...

Reply:
I removed this method exactly for that reason already. The implementation of the SRS use cases is already in the same test class:
* shouldThrowExceptionWhenValidateForSrsIsOnTheSameWeek
* shouldNotThrowExceptionWhenOnlyUpdateWeekIsTheSameSrs